provider/openstack/firewaller.go

```diff
@@ -73,6 +73,13 @@ type Firewaller interface {
 	// If the model security group doesn't exist, return a NotFound error
 	ModelIngressRules(ctx context.ProviderCallContext) (firewall.IngressRules, error)
 
+	// DeleteMachineGroup deletes the security group unique to a given machine.
+	// If we're in firewall mode 'instance' there will be only one, the security
+	// group used to configure this specific instance. The lifecyle of sec groups
```

Comment:
**todo;** I think you mess up your commit :D

Doesn't look informative to me... Not even english. Maybe it's british to initiali**s**e a sentence without finishing it ? :p 
 And `lifecyle` stand for `lifecycle`, maybe ?

Reply:
Oops, seems I rushed it a little :sweat_smile:

Reply:
Ok, third time lucky